extensions/sql-database-projects/src/dialogs/addDatabaseReferenceDialog.ts

```diff
@@ -291,10 +299,61 @@ export class AddDatabaseReferenceDialog {
 		};
 	}
 
+	private createSystemDbReferenceTypeRadioButtons(): azdataType.FormComponent {
+		this.systemDatabasePackageRefRadioButton = this.view!.modelBuilder.radioButton()
+			.withProps({
+				name: 'systemDbRefType',
+				label: constants.packageReference
+			}).component();
+
+		this.systemDatabasePackageRefRadioButton.onDidChangeCheckedState((checked) => {
+			if (checked) {
+				this.systemDbRefType = SystemDbReferenceType.PackageReference;
+			}
+		});
+
+		this.systemDatabaseArtifactRefRadioButton = this.view!.modelBuilder.radioButton()
+			.withProps({
+				name: 'systemDbRefType',
+				label: constants.artifactReference
+			}).component();
+
+		this.systemDatabaseArtifactRefRadioButton.onDidChangeCheckedState((checked) => {
+			if (checked) {
+				this.systemDbRefType = SystemDbReferenceType.ArtifactReference;
+			}
+		});
+
+		const radioButtons = [this.systemDatabasePackageRefRadioButton!, this.systemDatabaseArtifactRefRadioButton!];
```

Comment:
Just a thought. 
Does it make sense to update the sequence of references to Artifact reference first and then package reference, since artifact was the default until now, and is for legacy projects? 
(But the current sequence makes sense too, since package reference is the default right now here.)

Reply:
This is only for SDK-style projects because legacy projects can only use Artifact references. I made Package reference first to be the "default" to help encourage people to use it, then eventually we can remove the radio buttons and continue with package reference as the default (previously discussed on a teams chat)

Reply:
Yep, makes sense.